node/send_msg_using_node/index.js

```diff
@@ -0,0 +1,26 @@
+// Include voyage module
+const Vonage = require('@vonage/server-sdk')
+
+const vonage = new Vonage({
+  apiKey: "0ab91e43",
+  apiSecret: "81fPXp0J9HbhSC8w"
+})
+
+// Initialize with sender and reciever
+// mobile number with text message
+const from = "Vonage APIs";
+const to = "+12345678";
+const text = 'Programming is fun!';
+
+
+vonage.message.sendSms(from, to, text, (err, responseData) => {
```

Comment:
The issue asks for a set of supported channels, request payload, and response. Please make sure to follow the instructions in the [issue](https://github.com/appwrite/appwrite/issues/3954).